src/pkcs11/framework-pkcs15.c

```diff
@@ -5575,6 +5658,11 @@ static CK_RV register_ec_mechanisms(struct sc_pkcs11_card *p11card, int flags,
 			return rc;
 	}
 
+#ifdef ENABLE_OPNSSL
+	/* if card does not support a HASH but supports RAW add split mech */
+
+#endif
+
```

Comment:
Whats point of this change? There is typo in `ENABLE_OPNSSL` and the ifdef contains only a comment.

Reply:
Aah, no point.  Removed in   https://github.com/OpenSC/OpenSC/pull/2190/commits/b7b10c027dd0f8f89e8ab8ca44f978b9481c8c84